core/src/main/resources/META-INF/services/io.grpc.NameResolverProvider

```diff
@@ -0,0 +1 @@
+io.grpc.internal.DnsNameResolverProvider
```

Comment:
When additional NameResolver are created, will they have to be added to this file? If so, why not use the order in this file as the priority?